girder_annotation/test_annotation/web_client_specs/annotationListSpec.js

```diff
@@ -374,8 +374,8 @@ describe('AnnotationListWidget', function () {
         it('test header permission editor link', function () {
             var $el = $('.g-annotation-list');
             expect($el.find('th .g-annotation-permissions').length).toBe(1);
-            $el.find('th.g-annotation-select input').click();
-            $el.find('.g-annotation-permissions').click();
+            $el.find('th .g-annotation-select input').click();
```

Comment:
You _don't_ want the space here.  The `.g-annotation-select` class is on the `th` element (whereas the `.g-annotations-permissions` class is on the control within its `th` element.